unit_test/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/../test_common)
 INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING ${CMAKE_CURRENT_SOURCE_DIR}/../test_common)
 
+# Kyungjoo: when kokkoskernels and kokkos are symbolic link and their src directories are overrided
```

Comment:
What does it mean for "users to put kokkoskernels and kokkos at the same place"?

Reply:
Thanks for explaining!